bin/nodeshift

```diff
@@ -53,6 +53,15 @@ yargs
     type: 'string',
     default: 'latest'
   })
+  .options('outputImageStream', {
+    describe: 'The name of the Image Stream to output to.  Defaults to project name from package.json',
+    type: 'string'
+  })
+  .options('outputImageStreamTag', {
+    describe: 'The Tag of the Image Stream to output to.',
```

Comment:
Minor nit: you don't need to capitalize `Tag` here. Also, here and other places, if you are going to capitalize "Image Stream", you might as well write it as the kube resource `ImageStream`.